.github/workflows/Example.yml

```diff
@@ -1,76 +1,103 @@
-name: Examples v1 (latest)
+name: Examples
 
 on:
-  workflow_run:
-    workflows: ["Test v1 (latest)"]
-    types:
-      - completed
+  workflow_dispatch:
+  pull_request:
   push:
-    branches:
-      - examples
-  
+    branches: 
+      - main
+    paths:
+      - 'src/**'
+      - 'examples/**'
+      - '.github/workflows/Example.yml'
+      - 'Project.toml'
+
 jobs:
-  sync-files:
-    if: |
-      ${{ (github.event.workflow_run.conclusion == 'success' && github.sha == github.event.workflow_run.head_sha) || 
-      (github.event_name == 'push' && github.event.head_commit.committer.name != 'github-actions[bot]') }}
+  jupyter:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
-        julia-version: ['1.9']
+        os: [windows-latest] # ubuntu-latest
+        file-name: []
+        julia-version: ['1.10'] # 1.9
```

Comment:
better `lts`

Reply:
impossible as discussed some time ago with FMI.jl: there is no jupyter notebook kernal named "julia-lts". but there is one named "julia-1.10". See line 46 for the occurance